packages/formspree-core/src/utils.ts

```diff
@@ -171,12 +171,15 @@ export const handleSCA = async ({
   }
 };
 
-export function handleLegacyErrorPayload({
-  body,
-  response
-}: SubmissionResponse): SubmissionResponse {
-  if (!hasErrors(body) && (body as any)?.error) {
-    body = { errors: [{ message: (body as any).error }] };
+export function handleLegacyErrorPayload(
+  response: SubmissionResponse
+): SubmissionResponse {
+  let body = response.body as any;
```

Comment:
Do we still need `as any` here? Also, `body` is not reassigned so we can use `const`.